PSChiaPlotter/Public/Get-ChiaKPlotCombination.ps1

```diff
@@ -1,31 +1,85 @@
 function Get-ChiaKPlotCombination{
+    <#
+        .SYNOPSIS
+            Display compatible combinations of plots to fill a plot.
+        
+        .DESCRIPTION
+            Display compatible combinations of plots to fill a plot.
+        
+        .PARAMETER FreeSpace
+            Specifies the parameter name.
+        
+        .PARAMETER DriveLetter
+            Specifies the extension. "Txt" is the default.
+        
+        .PARAMETER upto
+            Only return combinations with this being the max size.
+            Options: 32, 33, 34 or 35 (default is all combinations)
+    
+        .PARAMETER c
+            Clears the screen first.
+            
+        .INPUTS
+            Inputs (if any)
+                    
+        .OUTPUTS
+            Displays a list of possible plot size combinations to maximize drive space.
+            
+        .EXAMPLE
+            PS> Get-ChiaKPlotCombination -FreeSpace 108.877420954
+            File.txt
+            
+        .EXAMPLE
+            PS> Get-ChiaKPlotCombination
+            File.doc
+            
+        .EXAMPLE
+            PS> Get-ChiaKPlotCombination
+            File.doc
+            
+        .LINK
+            PowerShell Gallery: https://www.powershellgallery.com/packages/PSChiaPlotter
+
+        .LINK
+            GitHub Project: https://github.com/MrPig91/PSChiaPlotter
+
+        .LINK
+            Set-Item
+    #>
     [CmdletBinding(DefaultParameterSetName = "DriveLetter")]
     param(
         [Parameter(ParameterSetName="FreeSpace")]
         [int64[]]$FreeSpace,
         [Parameter(ParameterSetName="DriveLetter")]
-        [string[]]$DriveLetter = (Get-Volume).DriveLetter
+        [string[]]$DriveLetter = (Get-Volume).DriveLetter,
+        [string]$UpTo = "K35",
```

Comment:
Can we do a validate range for this parameter? I think it will ensure proper values get inputted.
[ValidateRange(32,35)]
[int]$UpTo = 35

If you want to keep it as a string with the letter K, perhaps we can [ValidateSet("K32","K33"...)]